src/Symfony/Bundle/FrameworkBundle/Resources/config/templating_php.xml

```diff
@@ -44,7 +44,7 @@
         </service>
 
         <service id="templating.asset.path_package" class="%templating.asset.path_package.class%" abstract="true">
-            <argument type="expression">service('request_stack').getMasterRequest()</argument>
+            <argument type="expression">service('request_stack').getMasterRequest().getBasePath()</argument>
```

Comment:
this is broken if the service gets instantiated before the request is set, which would not allow us to get rid of the request scope in the container (children of this abstract service definition are set to the request scope currently). It might be worth a bigger refactoring.